transposon/genotypeClones.py

```diff
@@ -441,14 +441,23 @@ def writeOutputFiles(G, clones, output, outputlong, outputwide, cloneobj, graphO
             umi_count = clone['umi_count']
             bcs = clone['bcs']
             cells = clone['cells']
+            transfection = set()
+            if transfectionKey is not None:
+                skip = set(["conflicting", "uninformative", "None"])
+                transfection = set(G.nodes[bc][transfectionKey] for bc in bcs if G.nodes[bc][transfectionKey] not in skip)
+            if len(transfection) == 0:
+                transfection.add("None")
```

Comment:
Andre, what is this doing here? Didn't the pruning ensure that each clone has just one transfection value?

Reply:
The command line above gathers the BCs transfection assignment when available.
You are right that prunning such as `--removeConflictingCells` and `--removeConflictingBCs` will avoid more than one transfection be present per clone, but they are optional parameters.

Reply:
take a look at the updated comment, is that accurate?

Reply:
Yes it is